build_defs/requirements.txt

```diff
@@ -20,3 +20,10 @@ mkdocstrings[python]
 firebase-admin
 matplotlib
 pandas
+sqlalchemy
```

Comment:
I don't think we should mark these as "Deps for examples" because you are actually loading them in Mesop code, but you're lazy loading them. I'd add a section above L11 like "Optional (lazily-loaded) deps:"

Reply:
Sure sounds good.